server/controllers/overviewController.ts

```diff
@@ -39,24 +48,30 @@ export default class OverviewController {
     private readonly visitsService: VisitsService,
     private readonly prisonerScheduleService: PrisonerScheduleService,
     private readonly incentivesService: IncentivesService,
+    private readonly userService: UserService,
+    private readonly personalPageService: PersonalPageService,
+    private readonly offenderService: OffenderService,
+    private readonly professionalContactsService: ProfessionalContactsService,
   ) {}
 
   public async displayOverview(req: Request, res: Response, prisonerData: Prisoner, inmateDetail: InmateDetail) {
     const { clientToken } = req.middleware
-    const { userRoles, caseLoads } = res.locals.user
-    const { prisonId, bookingId, prisonerNumber } = prisonerData
+    const { userRoles, caseLoads, staffId } = res.locals.user
+    const { prisonId, bookingId, prisonerNumber, prisonName } = prisonerData
+    const { courCasesSummaryEnabled } = config.featureToggles
+
+    const activeCaseloadId = caseLoads.find((caseload: CaseLoad) => caseload.currentlyActive)?.caseLoadId
     const belongsToCaseLoad = prisonerBelongsToUsersCaseLoad(prisonId, caseLoads)
     const isPomOrReceptionUser = userHasRoles([Role.PomUser, Role.ReceptionUser], userRoles)
     const isGlobalSearchUser = userHasRoles([Role.GlobalSearch], userRoles)
+    const isYouthPrisoner = youthEstatePrisons.includes(prisonerData.prisonId)
```

Comment:
`prisonId` has already been destructured earlier so can just use that "standalone"